src/libical-1-fixes.patch

```diff
@@ -0,0 +1,133 @@
+This file is part of MXE.
+See index.html for further information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Wed, 13 Apr 2016 20:54:43 +0300
+Subject: [PATCH] fix definitions of gmtime/gmtime_r
+
+Fix the following error:
+config.h:508:37: error: expected initializer before '?' token
+
+The error is caused by the following combination:
+
+ #include <unistd.h>
+ #define gmtime_r(tp,tmp) ((gmtime(tp))?(*(tmp)=*(gmtime(tp)),(tmp)):0)
+ #include <time.h>
+
+time.h declares and defines gmtime_r. Because of the macro, the definition
+of gmtime_r is broken. This commit moves time.h before the macro define.
+
+diff --git a/config.h.cmake b/config.h.cmake
+index 1111111..2222222 100644
+--- a/config.h.cmake
++++ b/config.h.cmake
+@@ -495,6 +495,7 @@ typedef ssize_t IO_SSIZE_T;
+ #endif
+ #endif
+ 
++#include <time.h>
+ /* gmtime_r - thread safe gmtime() really only needed on Unix */
+ #if !defined(HAVE_GMTIME_R)
+ #if !defined(_WIN32)
+@@ -507,7 +508,6 @@ typedef ssize_t IO_SSIZE_T;
+ /* FYI: The gmtime() in Microsoft's C library is MT-safe */
+ #define gmtime_r(tp,tmp) (gmtime(tp)?(*(tmp)=*gmtime(tp),(tmp)):0)
+ #endif
+-#include <time.h>
+ 
+ /* localtime_r - thread safe localtime() really only needed on Unix */
+ #if !defined(HAVE_LOCALTIME_R)
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Thu, 14 Apr 2016 01:33:47 +0300
+Subject: [PATCH] test, examples: fix linking with static library
```

Comment:
It is a useful option for upstream, but we should not use it in MXE, because building tests and examples serves as a test for linking errors.